concepticondata/conceptlists.tsv

```diff
@@ -330,3 +330,5 @@ Baroni-2011-200	Baroni, Marco and Lenci, Alessandro	2011		200	relations	English
 Hill-2015-999	Hill, Felix and Reichart, Roi and Korhonen, Anna	2015		999	relations	English	English	https://fh295.github.io/simlex.html	Hill2015		SimLex999 is a set of 999 associations between two word pairs containing similarity annotations and concreteness values for each word pair. The word pairs were split into two lines to make a mapping to the Concepticon concepts possible. Note that the order is only relevant to the column ASSOCIATION_STRENGTH.		SimLex999
 Luangthongkum-2019-341	Luangthongkum, Theraphan	2019		341	questionnaire, areal	English	Karenic varieties spoken in Thailand	https://evols.library.manoa.hawaii.edu/bitstream/10524/52441/D01Luangthongkum2019view.pdf	Luangthongkum2019		The given 341-item list was created in the reconstruction of the phonology and lexicon of Proto-Karen, based on the ten selected Karenic varieties spoken in Thailand. The data for the analysis comes from the author’s fieldwork notes, made in the North and West of Thailand.	xvi-xliv	
 Pepper-2020-100	Pepper, Steve	2020		100	specific	English	Global		Pepper2020		The items in this list were used for a typological investigation of the morphology and semantics of binominal lexemes, i.e. of concepts typically expressed by nominal compounds, across 106 different languages.	393-394	
+PharaoHansen-2020-100	Pharao Hansen, Magnus	2020		100	basic, areal	Spanish	Southern Uto-Aztecan languages		Pharao2020		The given 100-item list was initially taken from [Miller-1984-100](:ref:Miller-1984-100) and was used in the preliminary analysis of the relationship between the six selected languages within the Southern Uto-Aztecan branch based on a wide range of empirical data.		
+Miller-1984-100	Miller, Wick R.	1984		100	basic, areal	Spanish	Uto-Aztecan languages		Miller1984	Miller1984	The current list represents a modified version of [Swadesh-1955-100](:ref:Swadesh-1955-100) with twelve substitutions (*belly button*, *heavy*, *old*, *rope*, *salt*, *sing*, *sky*, *snake*, *snow*, *vomit*, *wind*, and *year* replace *all*, *full*, *I*, *round*, *say*, *swim*, *that*, *this*, *thou*, *we*, *what*, and *who*). It was used in the reconstruction of the Uto-Aztecan language tree based on the investigation of cognate sets in the selected thirty-two languages.		
```

Comment:
One question left: "belly button": is this a NEW concpept or an old one? I guess a new one, right? In general, although I appreciate the detail, it may be useful to not list the substitutions, maybe only list one or two as exemples, as one can compare them anyway, and if so, I think it is also good to list the concepticon gloss, so it is clearer, which item we talk about.

Reply:
The gloss 'belly button' is usually mapped to 1838 NAVEL. 

I think it's interesting that they did not include 'I' and 'thou' which are very frequently used in other lists and added glosses such as 'snow' instead. @Kristina-Pianykh you can of course also pick out other examples.